modules/claims_api/spec/models/auto_establish_claim_spec.rb

```diff
@@ -307,200 +307,38 @@
       end
```

Comment:
This file looks worse than it actually is.
All I've done in this file is remove all of the old specs that checked the old `serviceBranch` mappings.
All of the tests are now covered in the new `service_branch_mapper_spec.rb` class. (see above)